tips/tests/test_rule_engine.py

```diff
@@ -239,3 +108,80 @@ def test_apply_rules_recursing(self):
         rules = [{"type": "ref", "ref_id": "1"}]
         with self.assertRaises(RecursionError):
             apply_rules(self.test_data, rules, compound_rules)
+
+    def test_stadspas(self):
+        fixture = get_fixture()
+        user_data = fixture
+        rules = [
+            {"type": "ref", "ref_id": "1"} # ID 1 is the stadspas rule
+        ]
+        self.assertTrue(apply_rules(self.user_data, rules, compound_rules))
+        # Change birth date so test will assert differently
+        user_data['brp']['persoon']['geboortedatum'] = '2012-01-01T00:00:00Z'
+        self.assertFalse(apply_rules(self.user_data, rules, compound_rules))
```

Comment:
Wat test je in deze test? het heet test_statspas, maar je veranderd iemands geboorte datum? Bedenk welke data moet veranderen om het zo te maken dat er getest word dat de rule niet True is. Dit geld ook voor alle volgende tests.